opium_kernel/src/opium_kernel.mli

```diff
@@ -213,6 +213,24 @@ module Server_connection : sig
     -> 'a Lwt.t
 end
 
+module Static : sig
+  val serve
+    :  read:(unit -> Rock.Body.t option Lwt.t)
+    -> ?mime_type:string
+    -> ?etag_of_fname:(string -> string)
+    -> ?headers:Rock.Headers.t
+    -> string
+    -> Rock.Handler.t
+
+  val m
+    :  read:(string -> Rock.Body.t option Lwt.t)
```

Comment:
I think it might be nicer if we have `read` have the type `string -> (Rock.Body.t, [Status.client_error | Status.server_error]) Lwt_result.t`.

Reply:
Done!